src/components/App.js

```diff
@@ -80,12 +89,13 @@ class App extends React.Component {
     let searches = {
       Venues: searchBars("", this.state.bars),
       Shows: searchShows("", this.state.bars),
-      Queens: searchQueens("", this.state.queens)
+      Queens: searchQueens("", this.state.queens),
+      Favorites: this.state.favoritesList || []
     };
     this.setState({
       searchResults: searches[type]
-    });
-  };
+    }, this.displayAllSearchResults());
```

Comment:
I assume the ` }, this.displ... ` is triggering our showAllResults function directly after setting the searchQuery to state?
@jacobogart

Reply:
Yes, I just piggybacked off the search Results functionality for all the hamburger menu functions